src/index.html

```diff
@@ -7,8 +7,49 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img
+        src="/images/logo.png"
+        alt="MOYO Logo"
+        class="logo__img">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">APPLE</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">SAMSUNG</a>
```

Comment:
No need type in uppercase here, as per requirements: `CSS is used to show all letters in Uppercase (don't type them in HTML)`

```suggestion
            <a class="nav__link" href="#">Samsung</a>
```